src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -213,6 +215,55 @@ def load_gen_kw(
             result.append(data)
         return np.stack(result).T
 
+    def save_surface_file(self, key: str, realization: int, file_name: str) -> None:
+        output_path = self.mount_point / f"surface-{realization}"
+        Path.mkdir(output_path, exist_ok=True)
+        surf = xtgeo.surface_from_file(file_name, fformat="irap_ascii")
+        surf.to_file(output_path / f"{key}.irap", fformat="irap_ascii")
+        self.getStateMap().update_matching(
+            realization,
+            RealizationStateEnum.STATE_UNDEFINED,
+            RealizationStateEnum.STATE_INITIALIZED,
+        )
+
+    def save_surface_data(
+        self,
+        key: str,
+        realization: int,
+        base_file_name: str,
+        data: "npt.NDArray[np.double]",
+    ) -> None:
+        output_path = self.mount_point / f"surface-{realization}"
+        Path.mkdir(output_path, exist_ok=True)
+        surf = xtgeo.surface_from_file(base_file_name, fformat="irap_ascii")
+        surf.set_values1d(data, order="F")
```

Comment:
Have not used xtgeo before.
Does this "F" refer to Fortran order?

Reply:
I suppose..  This is up for debate, but it was due to be inline with ecl version of surface :shrug:

Reply:
Alright 👍 
This looks good to me, but I think someone with a bit more context should also have a look.